app/middleware/csp.js

```diff
@@ -7,9 +7,45 @@ const environment = process.env.ENVIRONMENT
 
 const sentryCspReportUri = `${cspReportUri}&sentry_environment=${environment}`
 
+// Worldpay 3ds flex iframe
+const frameSource = ["'self'", 'https://secure-test.worldpay.com/']
+
+// Google analytics
+const imgSource = ["'self'", 'https://www.google-analytics.com/', 'http://www.google-analytics.com/']
+
+// Google analytics
+const scriptSource = ["'self'", 'https://www.google-analytics.com/', 'http://www.google-analytics.com/',
+  "'unsafe-inline'"]
```

Comment:
`unsafe-inline` is temporary pending externalising frontend scripts. This will let us test all external rules and then restrict scripts and apply non-report only once they have been moved out.